includes/shortcodes.php

```diff
@@ -699,6 +708,16 @@ function give_totals_shortcode( $atts ) {
 	<?php
 	$give_totals_output = ob_get_clean();
 
+	/**
+	 * Give Action fire after the total goal shortcode rendering is end.
+	 *
+	 * @since 2.1.4
+	 *
+	 * @param array $atts shortcode attribute.
+	 * @param HTML $give_totals_output shortcode output.
+	 */
+	do_action( 'give_totals_goal_shortcode_end', $atts, $give_totals_output );
```

Comment:
@raftaar1191  rename it to `give_totals_goal_shortcode_after_render`

Reply:
Changed @ravinderk